common/containers/Tabs/SendTransaction/components/ConfirmationModal.jsx

```diff
@@ -16,12 +18,14 @@ type Props = {
   rawTransaction: RawTransaction,
   wallet: BaseWallet,
   node: NodeConfig,
+  token: ?Token,
   onConfirm: RawTransaction => void,
   onCancel: () => void
 };
 
 type State = {
```

Comment:
You def should recalculate on every render. It is never too slow and when it is shouldComponentUpdate can ensure you only rerender when props change.